recipes/cairo/meson/conanfile.py

```diff
@@ -224,166 +232,120 @@ def package(self):
             meson = self._configure_meson()
             meson.install()
         self._fix_library_names()
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "*.pdb")
+        files.rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        files.rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
 
     def package_info(self):
-        self.cpp_info.components["cairo_"].names["pkg_config"] = "cairo"
+        base_requirements = {"pixman::pixman"}
+        base_system_libs = {}
+
+        def add_component_and_base_requirements(component, requirements, system_libs=None):
+            self.cpp_info.components[component].set_property("pkg_config_name", component)
```

Comment:
please leave the old `.names["pkg-config"]` for the v1 generators (we have not started removing them)